autogpt_platform/backend/backend/blocks/example/example.py

```diff
@@ -0,0 +1,143 @@
+import logging
+from typing import Any
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema, BlockType
+from backend.data.model import (
+    APIKeyCredentials,
+    ContributorDetails,
+    CredentialsField,
+    SchemaField,
+)
+
+from ._api import ExampleClient
+from ._auth import TEST_CREDENTIALS, TEST_CREDENTIALS_INPUT, ExampleCredentialsInput
+
+logger = logging.getLogger(__name__)
+
+
+class ExampleBlock(Block):
+
+    class Input(BlockSchema):
+        name: str = SchemaField(
+            description="The name of the example block", placeholder="Enter a name"
+        )
+        greetings: list[str] = SchemaField(
```

Comment:
It's good to show features around the field like `advanced` or `hidden` toggle

Reply:
likewise the dependent fields with descriminators

Reply:
What exactly do advanced and hidden do?

I think they just hide if it is shown in the UI - is this correct?

I've added examples of them